node/configuration.yaml

```diff
@@ -14945,6 +14945,51 @@ mainnet_launch_base:
 
 ##############################################################################
 ##                                                                          ##
+##   Devnet config                                                          ##
+##                                                                          ##
+##############################################################################
+
+devnet: &devnet
+  <<: *mainnet_base
+  core:
+    <<: *mainnet_base_core
+    genesis:
+      <<: *mainnet_base_genesis
+      spec:
+        <<: *mainnet_base_spec
+        blockVersionData:
+          <<: *mainnet_base_blockVersionData
+          slotDuration:      20000 # set to desired value
+        protocolConstants:
+          <<: *mainnet_base_protocolConstants
+          protocolMagic: 170987650
+          k: 6 # set to desired value
+        avvmDistr: *mainnet_avvmDistr
+        initializer:
+          testnetInitializer:
+            testBalance:
+              poors:        200 # should be enough for developers
+              richmen:      4   # this is the number of core nodes,
```

Comment:
@gromakovsky, can we reduce that to `3`, to make the dev clusters lighter?

Reply:
I've set it to 3, please re-approve.